internal/selectors/selectors.go

```diff
@@ -161,6 +161,9 @@ func GetNoneSelector(inst *unstructured.Unstructured) (bool, error) {
 // cmExclusionsByName are known (istio and kube-root) CA configmap which are excluded from propagation
 var cmExclusionsByName = []string{"istio-ca-root-cert", "kube-root-ca.crt"}
 
+// cmExclusionsByName are known (istio and kube-root) CA configmap which are excluded from propagation
```

Comment:
Fix comment

Reply:
no , openshift does not set labels on it
here is my output
```
oc describe rolebinding -n abc       
Name:         postgresql-operator.v0.1.1
Labels:       <none>
Annotations:  <none>
Role:
  Kind:  Role
  Name:  postgresql-operator.v0.1.1
Subjects:
  Kind            Name                 Namespace
  ----            ----                 ---------
  ServiceAccount  postgresql-operator  


Name:         system:deployers
Labels:       <none>
Annotations:  openshift.io/description:
                Allows deploymentconfigs in this namespace to rollout pods in this namespace.  It is auto-managed by a controller; remove subjects to disa...
              propagate.hnc.x-k8s.io/none: true
Role:
  Kind:  ClusterRole
  Name:  system:deployer
Subjects:
  Kind            Name      Namespace
  ----            ----      ---------
  ServiceAccount  deployer  abc


Name:         system:image-builders
Labels:       <none>
Annotations:  openshift.io/description:
                Allows builds in this namespace to push images to this namespace.  It is auto-managed by a controller; remove subjects to disable.
              propagate.hnc.x-k8s.io/none: true
Role:
  Kind:  ClusterRole
  Name:  system:image-builder
Subjects:
  Kind            Name     Namespace
  ----            ----     ---------
  ServiceAccount  builder  abc


Name:         system:image-pullers
Labels:       <none>
Annotations:  openshift.io/description:
                Allows all pods in this namespace to pull images from this namespace.  It is auto-managed by a controller; remove subjects to disable.
              propagate.hnc.x-k8s.io/none: true
Role:
  Kind:  ClusterRole
  Name:  system:image-puller
Subjects:
  Kind   Name                        Namespace
  ----   ----                        ---------
  Group  system:serviceaccounts:abc  
```